client/my-sites/themes/themes-selection.jsx

```diff
@@ -144,6 +175,8 @@ const ConnectedThemesSelection = connect(
 	( state, { filter, page, search, tier, vertical, siteId, queryWpcom } ) => {
 		const isJetpack = isJetpackSite( state, siteId );
 		const siteIdOrWpcom = ( siteId && isJetpack && ! ( queryWpcom === true ) ) ? siteId : 'wpcom';
+		// If a preview action is passed, use that. Otherwise, use our own.
+		// we switch action for one that will tell preview that theme comes from wpocm list
```

Comment:
Ahh it should be totally removed, I was here together with some logic I have copied. I have removed the logic and forgot about the comments.